content/md/cloud/2-access-flexibility-server.md

```diff
@@ -40,6 +43,39 @@ Once the contacts are created and their keys imported, the project manager can a
 The SSH key access is **not instantaneous**. Initially, the status will be `Creation pending` and then will change to `Activated` once the access is given.
 :::
 
+### You can add access to your flexibility instance to a contact from another company.
+
+1. Click on the `Add contact key` button
+1. Select `Select other contact`
+1. Enter the email of the contact, `This contact exists in the portal.` must appear.
+1. Enter the name of the key and the public key.
+1. Click on the `Add contact key` button to validate the creation.
+
+![Add existing contact from other company](../img/add_contact_other_company_flex.png)
+
+### You can directly create a contact for your company or as an external contact and give him access to the enterprise repository.
+
+::: info
```

Comment:
"Info" should be used to give complementary information, and not the main content of the section.